README.md

```diff
@@ -8,27 +8,307 @@
+
+The Orderbook SDK package provides a set of functions and utilities for interacting with the Immutable zkEVM Orderbook. It offers a simple and easy-to-use interface for querying orderbook data, such as orders and trades. 
+
+The Orderbook SDK package is built using TypeScript and Rollup. The package is exposed to the SDK package that is published to NPM.
+
+More information can be found in the README file in the Orderbook folder [here](/packages//orderbook//README.md).
+
+### Passport
+
+The Passport SDK folder contains both the Passport SDK package, as well as the Passport Sample App. The Passport SDK package provides a set of functions and user interfaces for interacting with Immutable Passport, a blockchain-based identity and wallet system.
+
+The Passport SDK package is built using TypeScript and Rollup. The package is exposed to the SDK package that is published to NPM.
+
+A sample app is also included in the Passport SDK folder to demonstrate how to use the Passport SDK package and for testing purposes. The sample app can be run in development mode to test changes and new features before deploying them to production.
+
+More information for the Passport SDK can be found in the README file in the Passport SDK folder [here](/packages//passport//sdk//README.md).
+
+More information for the Passport Sample App can be found in the README file in the Passport Sample App folder [here](/packages//passport//sdk-sample-app//README.md).
+
+### Webhook
+
+The Webhook package consists of a utility function, `handle`, that can be used to work with Immutable webhooks. The `handle` function is used to verify and process incoming webhook requests from Immutable services, such as the Immutable Marketplace. It provides a simple and easy-to-use interface for handling webhooks.
+
+The Webhook package is built using TypeScript and Rollup. The package is exposed to the SDK package that is published to NPM.
+
+More information can be found in the README file in the Webhook folder [here](/packages//webhook//sdk//README.md).
+
+### X-Client
+
+The X-Client SDK package provides a set of functions and utilities for interacting with the ImmutableX StarkEx-based Layer 2 blockchain. It provides an `IMXClient` that is used for this purpose. 
+
+The X-Client SDK package is built using TypeScript and Rollup. The package is exposed to the SDK package that is published to NPM.
+
+### X-Provider
+
+The X-Provider SDK package provides methods that implement the signable actions with IMX only. 
+
+The X-Provider SDK package is built using TypeScript and Rollup. The package is exposed to the SDK package that is published to NPM.
+
+More information can be found in the README file in the X-Provider folder [here](/packages//x-provider//README.md).
+
 # Contribution Guides
 See [CONTRIBUTING.md](https://github.com/immutable/ts-immutable-sdk/blob/main/CONTRIBUTING.md)
 
 Internal maintainers' guide is available at Immutable's wiki `UnifiedSDK Internal Development Guide`
```

Comment:
I changed title of the page to remove the word `unifiedsdk`
could you pls also update here? thank you!

Reply:
Updated 😄